epinorm/norm.py

```diff
@@ -1,40 +1,138 @@
 import json
 import pandas as pd
+import numpy as np
 import re
-
+import itertools
 from csv import QUOTE_NONE
-from epinorm.config import REF_DATA_DIR
+from transliterate import translit
+from transliterate.exceptions import LanguageDetectionError
+
 from epinorm.geo import NominatimGeocoder
-from epinorm.utils import coalesce
+from epinorm.config import MIN_ADMIN_EXCEPTIONS, REF_DATA_DIR, COUNTRIES_DATA, COUNTRIES_EXCEPTIONS, ADMIN_LEVELS_DATA
```

Comment:
Use a multi-line import statement:

```suggestion
from epinorm.config import (
    MIN_ADMIN_EXCEPTIONS,
    REF_DATA_DIR,
    COUNTRIES_DATA,
    COUNTRIES_EXCEPTIONS,
    ADMIN_LEVELS_DATA,
)
```